packages/stacked/lib/src/base_view_models.dart

```diff
@@ -498,3 +509,8 @@ class StreamData<T> extends _SingleDataSourceViewModel<T> {
     super.dispose();
   }
 }
+
+/// Interface: Additional actions that should be implemented by spcialised ViewModels
+abstract class IAdditionalSetup {
```

Comment:
None of the interfaces use the 'I' naming convention. Please change this to Initialisable instead of Additional setup. This will align the name of the function and the interface name.

Reply:
Renamed the interface and updated the same for the classes that implements this interface.